functional_unit_testing/parteh/PartehDriver.py

```diff
@@ -0,0 +1,770 @@
+# =======================================================================================
+#
+## @package PARTEH (Plant Allocatoin and Reactive Transport Exensible Hypotheses
```

Comment:
are all the files in the functional_unit_testing directory architecture around the standalone running of PARTEH? If so, I'm tempted to skip them to concentrate on the code that runs by default. Let me know if you'd like me to take a look at those as well.

Reply:
I would recommend that you skip everything in that directory, like you say.  These do not affect FATES, they are just used for testing, and they will be a large time-sink.